### PR TITLE
✨ Add additonal OpenAPI metadata parameters to `FastAPI` class, shown on the automatic API docs UI

### PR DESCRIPTION
**Pull request by dkreeft**
As described in #843 that I needed and stumbled upon. This PR adds extra metadata parameters to the code and docs. So now you can specify contact and license information as well as terms of service.

I tried to keep the naming of the parameters in line with the [specification](https://swagger.io/specification/), but unfortunately `license` is a Python built-in, so I went with `license_info` instead.

### CODE REVIEW

File: tests/test_tutorial/test_metadata/test_tutorial001.py [hunk 1]
```diff
@@ -7,9 +7,19 @@
 openapi_schema = {
     "openapi": "3.0.2",
     "info": {
-        "title": "My Super Project",
-        "version": "2.5.0",
-        "description": "This is a very fancy project, with auto docs for the API and everything",
+        "title": "My API",
+        "version": "0.0.1",
+        "description": "This is my API, with auto docs for the API and everything",
```
Comment (Kludex) [inline tests/test_tutorial/test_metadata/test_tutorial001.py:12]: Is there a reason for changing the fields `title`, `version` and `description` here?

File: fastapi/applications.py [hunk 2]
```diff
@@ -36,6 +36,9 @@ def __init__(
         title: str = "FastAPI",
         description: str = "",
         version: str = "0.1.0",
+        terms_of_service: Optional[str] = "",
+        contact: Optional[Dict[str, Union[str, Any]]] = None,
```
Comment (Kludex) [inline fastapi/applications.py:40]: Don't we need to validate the `keys` on the `contact` and `license_info`?

File: tests/test_tutorial/test_metadata/test_tutorial001.py (hunk 1 above)
Comment (dkreeft) [reply tests/test_tutorial/test_metadata/test_tutorial001.py:12]: Yes, the title and description did not reflect what they should be (`My Super Project`) imo.

File: fastapi/applications.py (hunk 2 above)
Comment (dkreeft) [reply fastapi/applications.py:40]: We could, but that is not really done for other arguments either.

File: fastapi/applications.py [hunk 3]
```diff
@@ -36,6 +36,9 @@ def __init__(
         title: str = "FastAPI",
         description: str = "",
         version: str = "0.1.0",
+        terms_of_service: Optional[str] = "",
```
Comment (Kludex) [inline fastapi/applications.py:39]:
```suggestion
        terms_of_service: str = "",
```

File: fastapi/openapi/utils.py [hunk 4]
```diff
@@ -330,13 +330,22 @@ def get_openapi(
     version: str,
     openapi_version: str = "3.0.2",
     description: str = None,
+    terms_of_service: str = None,
```
Comment (Kludex) [inline fastapi/openapi/utils.py:333]:
```suggestion
    terms_of_service: Optional[str] = None,
```

**Comment by dkreeft** [comment]
@Kludex do you mind approving this PR? Thanks!

**Comment by dkreeft** [comment]
@tiangolo any ETA for approving this request? In my team we are waiting for this change.

**Comment by tiangolo** [comment]
Thanks @dkreeft! :rocket: 

I updated it a bit, updated the type annotations, order of arguments, and the docs.

Thanks for your contribution! :coffee: :cake: 

This will be available later today in FastAPI version `0.68.0` 🔖🎉

**Pull request merged**